docs/Nerves Cheatsheet.cheatmd

```diff
@@ -0,0 +1,160 @@
+# Nerves User Guide
+
+## Overview
+
+### Terminology
+
+| Term             | Definition                 |
+| ---              | ---                        |
+| host             | The computer on which you are editing source code, compiling, and assembling firmware |
+| target           | The computer that runs the compiled code such as a Raspberry Pi Zero, Beaglebone, or other board |
+| toolchain        | The tools required to build code for the target, such as compilers, linkers, binutils, and C runtime |
+| system           | A lean Buildroot-based Linux root filesystem that has been customized and cross-compiled for a particular target |
+| firmware bundle  | A single file that contains the system, application, configuration and anything else needed to initialize and update the non- volatile storage of a device |
+| firmware image   | A large binary file generated from the firmware bundle that is a bit- for-bit image of a devices non- volatile memory |
+
+### Official targets
+{: .list-3}
+
+* Beaglebone Black and variants - `bbb`
+* Raspberry Pi Zero - `rpi0`
+* Raspberry Pi Zero W 2/Model 3A - `rpi3a`
+* Raspberry Pi 3B - `rpi3`
+* Raspberry Pi 4 - `rpi4`
+* GRiSP2 - `grisp2`
+* OSD32MP1 - `osd32mp1`
+* Generic x86_64 - `x86_64`
+
+## Getting started
+{: .col-2}
+
+### Install Nerves bootstrap archive
+
+```sh
+mix archive.install hex nerves_bootstrap
+```
+
+### Create a new Nerves project
+
+```sh
+mix nerves.new my_project
+cd my_project
+export MIX TARGET=<mix target>
+mix deps.get
+mix firmware
+```
+
+### Burn a MicroSD card for the first time
+
+Build the project and write it to a MicroSD:
+
+```sh
+mix firmware.burn
+```
+
+Write the most recently built firmware to MicroSD:
+
+```sh
+mix burn
+```
+
+### Update firmware via ssh
+
+```sh
+mix firmware.push [hostname or IP address]
```

Comment:
```suggestion
mix upload [hostname or IP address]
```
Is this still I think? I thought we removed `firmware.push`

Reply:
lol - I was copy/pasting from an old Nerves PDF cheatsheet and missed that.